src/pages/about/TeamSection.js

```diff
@@ -1,36 +1,42 @@
 import React from "react";
-import { teamImages, contributorImages } from "./aboutHelper";
+import Accordion from "../../components/accordion/accordion";
+
+import { boardOFDirectors, teamLeads, uxDesigners, frontEnd, dataScience, webGL, writers } from "../../pages/about/aboutHelper";
 
 /**
  * @returns {JSX.Element}
  * @constructor
  */
 
- export default function Team() {
+export default function Team() {
+
+	const {title:boardTitle, members:boardMembers} = boardOFDirectors;
+	const {title:teamLeadTitle, members:teamLeadmembers} = teamLeads;
+	const {title:uxTitle, members:uxMembers} = uxDesigners;
+	const {title:frontEndTitle, members:frontEndMembers} = frontEnd;
+	const {title:dataScienceTitle, members:dataScienceMembers} = dataScience;
+	const {title:webGLTitle, members:webGLMembers} = webGL;
+	const {title:writersTitle, members:writersMembers} = writers;
+
+	const boardOFDirectorsAccordion = <Accordion title={boardTitle} members={boardMembers}/>
+	const teamLeadAccordion = <Accordion title={teamLeadTitle} members={teamLeadmembers}/>
+	const uxDesignersAccordion = <Accordion title={uxTitle} members={uxMembers}/>
+	const frontEndAccordion = <Accordion title={frontEndTitle} members={frontEndMembers}/>
+	const dataScienceAccordion = <Accordion title={dataScienceTitle} members={dataScienceMembers}/>
+	const webGLAccordion = <Accordion title={webGLTitle} members={webGLMembers}/>
+	const writersAccordion = <Accordion title={writersTitle} members={writersMembers}/>
```

Comment:
This entire block of variable assignments is unnecessary. Instead of rendering `{boardOfDirectorsAccordion}` just directly render `<Accordion title={boardTitle} members={boardMembers}/>` in the `return`.